front/app/containers/Admin/formBuilder/components/FormResults/index.tsx

```diff
@@ -131,46 +129,16 @@ const FormResults = ({ intl: { formatMessage } }: WrappedComponentProps) => {
             { question, inputType, answers, totalResponses, required },
             index
           ) => {
-            const inputTypeText = get(messages, inputType, '');
-            const requiredOrOptionalText = required
-              ? formatMessage(messages.required)
-              : formatMessage(messages.optional);
-            const inputTypeLabel = `${formatMessage(
-              inputTypeText
-            )} - ${requiredOrOptionalText.toLowerCase()}`;
-
             return (
-              <Box
+              <FormResultsQuestion
                 key={index}
-                data-cy={`e2e-${snakeCase(question[locale])}`}
-                mb="56px"
-              >
-                <Title variant="h3" mb="0">
-                  <T value={question} />
-                </Title>
-                {inputTypeText && (
-                  <Text variant="bodyS" color="textSecondary" mb="0">
-                    {inputTypeLabel}
-                  </Text>
-                )}
-                {answers.map(({ answer, responses }, index) => {
-                  const percentage =
-                    Math.round((responses / totalResponses) * 1000) / 10;
-
-                  return (
-                    <CompletionBar
-                      key={index}
-                      bgColor={colors.primary}
-                      completed={percentage}
-                      leftLabel={answer}
-                      rightLabel={formatMessage(messages.choiceCount, {
-                        choiceCount: responses,
-                        percentage,
-                      })}
-                    />
-                  );
-                })}
-              </Box>
+                locale={locale}
```

Comment:
Would probably just use `useLocale` inside of `FormResultsQuestion` instead

Reply:
Did consider that, but thought it might take less resource to only call it once instead of once per question. Although probably neglible.